apim-operator/build/controller_resources/dockerFile.gotmpl

```diff
@@ -0,0 +1,46 @@
+#This image will be used by Kaniko pod to create the micro-gw image
```

Comment:
Let's add the license header.. For all the files except gen files, please add the license header

Reply:
added